### PR TITLE
enable travis; minor test-related fixes

### LINKED ISSUE #29: Set up Travis

### PR DESCRIPTION
**Pull request by smith**
The other changes were related to failures I was getting when running specs locally, but shouldn't affect anything else.

Fixes #29.

### CODE REVIEW

File: lib/tasks/travis-ci.rake [hunk 1]
```diff
@@ -0,0 +1,7 @@
+namespace :travis do
+  desc "Run all specs in except for the request specs"
+  RSpec::Core::RakeTask.new(:ci => :environment) do |task|
+    Rails.env = "test"
```
Comment (cnunciato) [inline lib/tasks/travis-ci.rake]: Just FYI, I had to set `RAILS_ENV=test` explicitly (or change this line to `ENV['RAILS_ENV']="test"`) in order to get `rake travis:ci` to pass.

**Comment by cnunciato** [comment]
Aside from my comment above, :+1:.

File: lib/tasks/travis-ci.rake (hunk 1 above)
Comment (smith) [inline lib/tasks/travis-ci.rake]: Really? I was having trouble with `bin/rake travis:ci` but not `rake travis:ci` or `bundle exec rake travis:ci` (which is what Travis actually uses.)

**Comment by chef-supermarket** [comment]
Hi. I am an automated pull request bot named Curry. There are commits in this pull request whose authors are not yet authorized to contribute to Chef Software, Inc. projects or are using a non-GitHub verified email address. To become authorized to contribute, you will need to sign the Contributor License Agreement (CLA) as an individual or on behalf of your company. [You can read more on Chef's blog.](https://www.chef.io/blog/2014/06/23/changes-to-the-contributor-license-agreement-process)

## Non-GitHub Verified Committers

There are 1 commit author(s) whose commits are authored by a non-GitHub verified email address. Chef will have to manually verify that they are authorized to contribute.

[Please sign the CLA here.](https://supermarket.chef.io/become-a-contributor)

**Pull request merged**